resolwe/process/fields.py

```diff
@@ -576,6 +585,8 @@ def __repr__(self):
 class DataField(Field):
     """Data object field."""
 
+    field_type = "data"
```

Comment:
(This should go into a separate commit.)

Reply:
Why? This enables the fix to work...

Reply:
:+1: